server/src/main/webapp/verifier.html

```diff
@@ -45,7 +50,16 @@ <h1 class="text-body-emphasis">Request Verified Identity Documents</h1>
                         Request PID (Full)
                     </button>
                 </div>
-
+            </div>
+            <div class="tab-pane fade" id="pills-pid_sdjwt" role="tabpanel" aria-labelledby="pills-pid_sdjwt-tab" tabindex="0">
+                <div class="d-grid gap-2 mx-auto">
+                    <button type="button" class="btn btn-primary btn-lg" onclick="requestDocument('pid_sdjwt_mandatory')">
+                        Request PID (Mandatory Elements)
```

Comment:
"Mandatory Claims", I think

Reply:
Should that also be the case for the two mdoc requests?

Reply:
for mdoc they are called "Data Elements" so "Mandatory Elements" is already right. So no action required here.